bin/bin.py

```diff
@@ -1,6 +1,11 @@
-from bottle import route, template
+from bottle import route, template, error, static_file
+from os import path
 from bin import root
 
+@route('/assets/<dir>/<filepath:path>')
+def styles(dir=None, filepath=None):
+    return static_file(path.join(dir, filepath), root=root.joinpath('assets'))

```

Comment:
No need for the `<dir>` placeholder, also why is the function named `styles` ?

    @route('/assets/<filepath:path>')
    def assets(filepath):
        return static_files(filepath, root=root / 'assets')